OpenRA.Mods.TS/Traits/NotifyActorsBelow.cs

```diff
@@ -0,0 +1,39 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.TS.Traits
+{
+	public interface INotifyActorAbove
+	{
+		void NotifyActorAbove(Actor self, Actor above);
+	}
+
+	public class NotifyActorsBelowInfo : TraitInfo<NotifyActorsBelow> { }
```

Comment:
Can we get a HashSet<string> of notification types here to avoid overlaps if something else wants to use this in the future?